recommender/engines/autoencoders/inference/embedding_component.py

```diff
@@ -22,11 +22,9 @@ def __init__(self):
     def __call__(self, use_cache=False, save_cache=True):
         """Performs embedding with normalization."""
 
-        print("Starting embedding...")
```

Comment:
Keeping those print statements makes more sense with the new commands [GH](https://github.com/cyfronet-fid/recommender-system/blob/54bd5a994b72705c78c6a10a9408e1578f9761d9/recommender/commands.py#L68). Am I right?